src/bootstrap/src/utils/exec.rs

```diff
@@ -120,7 +120,7 @@ impl BootstrapCommand {
         Self { failure_behavior: BehaviorOnFailure::DelayFail, ..self }
     }
 
-    #[must_use]
+    #[allow(dead_code)]
```

Comment:
Question [DEADCODE 1/2]: why do we change `#[must_use]` -> `#[allow(dead_code)]` *here*? If we need to suppress `#[must_use]` we could suppress at use site with like `let _ = ...;` right?

Reply:
These functions are not used at all (but still worth to keep) and causing warnings.